src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Generated/Models/Page2.cs

```diff
@@ -28,10 +28,10 @@ public class Page2<T> : IPage<T>
          /// <summary>
          /// Gets the link to the next page.
          /// </summary>
-         [JsonProperty("")]
+         [JsonProperty("NextLink")]
```

Comment:
This is auto generated from swagger spec and which in turn reflects field name directly from service data contracts.